### PR TITLE
Fix #8 add an option to skip tests

### LINKED ISSUE #8: Add an option to skip tests
We want this probably both per srcdeps scm repository and per all srcdeps. We should also add `srcdeps.maven.test.skip` and `srcdeps.skipTests`

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
@jpkrohling would you please review? I admit that I had no time to write tests for the new skipTests functionality.

File: srcdeps-maven-plugin/src/main/java/org/l2x6/maven/srcdeps/SrcdepsInstaller.java [hunk 1]
```diff
@@ -81,10 +106,14 @@ protected void checkout(DependencyBuild depBuild) throws MavenExecutorException
...
-        String srcAbsPath = checkoutDir.getAbsolutePath();
-        ScmVersion scmVersion = depBuild.getScmVersion();
-        String args = "-DcheckoutDirectory=" + srcAbsPath + " -DconnectionUrl=" + depBuild.getUrl()
-                + " -DscmVersionType="+ scmVersion.getVersionType() +" -DscmVersion=" + scmVersion.getVersion();
+        final ScmVersion scmVersion = depBuild.getScmVersion();
+
+        final String args = new ArgsBuilder().prop("checkoutDirectory", checkoutDir.getAbsolutePath())
+                .prop("connectionUrl", depBuild.getUrl()).prop("scmVersionType", scmVersion.getVersionType())
+                .prop("scmVersion", scmVersion.getVersion()).nonDefaultProp("skipTests", depBuild.isSkipTests(), false)
+                .nonDefaultProp("maven.test.skip", depBuild.isMavenTestSkip(), false).build();
```
Comment (jpkrohling) [inline srcdeps-maven-plugin/src/main/java/org/l2x6/maven/srcdeps/SrcdepsInstaller.java:114]: Why is there a `maven.test.skip` and a `skipTests`? Aren't they the same?

**Comment by jpkrohling** [comment]
Other than the comment about the `skipTests` and `maven.test.skip`, it looks good to me.

**Comment by ppalaga** [comment]
`skipTests` skips the execution of the tests only, `maven.test.skip` also skips the compilation of the tests http://maven.apache.org/surefire/maven-surefire-plugin/examples/skipping-test.html

**Pull request merged**